product_docs/docs/migration_portal/4/04_mp_migrating_database/01_mp_schema_extraction/known_issues_notes.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: "Known issues and additional notes"
```

Comment:
Some of the items in this section are limitations and not necessarily known issues.  I suggest changing the title of the section to "Known issues, limitations, and additional notes" or "Known isues, limitations, and notes".

Reply:
Renamed to Known issues, limitations, and notes